website/docs/tutorials/managing-access.md

```diff
@@ -123,19 +123,150 @@ const accessByAgent = unstable_getAgentAccessAll(litDatasetWithAcl);
+`null` if it does not exist.
+
+If it exists, that's great, and you can move on to the next section. You can create a new ACL if it
+does not exist yet, but be aware that doing so will result in any access that is currently granted
+to the Resource via the default access rules of its fallback ACL, as described in the previous
+section, will no longer apply.
+
+A new empty ACL can be initialised using
+[`unstable_createAcl`](api/modules/_acl_acl_#unstable_createacl), given that the current user has
+the rights to do so. However, keep in mind that this ACL will override any ACL that currently
+applies to it. The consequence is that if you do not give someone
+[Control](../glossary#control-access) access, nobody will ever be able to modify the ACL ever again.
+
+If the fallback ACL is available to the current user, it is possible to copy the currently
+applicable rules to a newly-initialised ACL; but be aware that future updates to the fallback ACL
+will not be reflected in the new ACL.
+
+The general process of changing access to a Resource is as follows:
+
+```typescript
+import {
+  unstable_fetchLitDatasetWithAcl,
+  unstable_hasResourceAcl,
+  unstable_hasFallbackAcl,
+  unstable_hasAccessibleAcl,
+  unstable_createAcl,
+  unstable_createAclFromFallbackAcl,
+  unstable_getResourceAcl,
+  unstable_setAgentResourceAccess,
+  unstable_saveAclFor,
+} from "@solid/lit-pod";
+
+// Fetch the LitDataset and its associated ACLs, if available:
+const litDatasetWithAcl = await unstable_fetchLitDatasetWithAcl(
+  "https://example.com"
+);
+
+// Obtain the LitDataset's own ACL, if available,
+// or initialise a new one, if possible:
+let resourceAcl;
+if (!unstable_hasResourceAcl(litDatasetWithAcl)) {
+  if (!unstable_hasAccessibleAcl(litDatasetWithAcl)) {
+    throw new Error(
+      "The current user does not have permission to change access rights to this Resource."
+    );
+  }
+  if (!unstable_hasFallbackAcl(litDatasetWithAcl)) {
+    throw new Error(
+      "The current user does not have permission to see who currently has access to this Resource."
+    );
+    // Alternatively, initialise a new empty ACL as follows,
+    // but be aware that if you do not give someone Control access,
+    // **nobody will ever be able to change Access permissions in the future**:
+    // resourceAcl = unstable_createAcl(litDatasetWithAcl);
+  }
+  resourceAcl = unstable_createAclFromFallbackAcl(litDatasetWithAcl);
+} else {
+  resourceAcl = unstable_getResourceAcl(litDatasetWithAcl);
+}
+
+// Give someone Control access to the given Resource:
+const updatedAcl = unstable_setAgentResourceAccess(
+  resourceAcl,
+  "https://some.pod/profile#webId",
+  { read: false, append: false, write: false, control: true }
+);
+
+// Now save the ACL:
+await unstable_saveAclFor(litDatasetWithAcl, updatedAcl);
+```
+
+### Setting public access
+
+:::note
+
+Unfortunately lit-pod does not yet support setting public access. Addition is planned, so please
+check back in the future.
```

Comment:
Just wondering if we could just state

lit-pod currently does not support setting public access.   See <some link to a public facing task list or whatnot>.

Reply:
Yes, that makes sense; best not to promise to avoid potentially over-promising. Unfortunately our issue tracker is private, so there's nowhere to point after "See ".

Fixed in 6ae01e1fcd35e8901601429c558fcc2c59723f1a.